src/server_manager/messages/en.json

```diff
@@ -105,6 +110,8 @@
   "notification-feedback-thanks": "Thanks for helping us improve! We love hearing from you.",
   "notification-key-added": "Key added",
   "notification-key-removed": "Key removed",
+  "notification-keys-port-saving": "Saving...",
+  "notification-keys-port-saved": "Saved",
```

Comment:
The keys for these could be more general (not specific to keys-port), i.e. `notificiation-saving`, `notification-saved`.

Reply:
Done